tests/Manifest1.jad

```diff
@@ -3,6 +3,6 @@ MIDlet-Jar-Size: 2436
 MIDlet-Jar-URL: tests.jar
 MIDlet-Name: AMIDlet
 MIDlet-Vendor: MIDlet Suite Vendor
-MIDlet-Version: 1.0.0
+MIDlet-Version: 1

```

Comment:
I've had to change this because there's a bug somewhere in our code that replaces "." characters with "/" characters in the args parameter.